packages/admin/cms-admin/src/form/file/FinalFormFileUpload.tsx

```diff
@@ -145,7 +153,7 @@ export const FinalFormFileUpload = <Multiple extends boolean | undefined>({
                 for (const file of acceptedFiles) {
                     const formData = new FormData();
                     formData.append("file", file);
-                    const response = await fetch(`${apiUrl}/file-uploads/upload`, {
+                    const response = await fetch(`${apiUrl}${uploadPath}`, {
```

Comment:
maybe the uploadPath prop should also include apiUrl - to allow uploading to an external api?

And maybe better naming alternative could be `uploadEndpoint`, `uploadApiPath` or `uploadApi` (I first thought this is about the path where the file is stored on the server (which file-uploads don't have))

Reply:
good idea, updated it to `uploadEndpoint`